tests/v2/validation/nodescaling/scale_replace_rke1_test.go

```diff
@@ -40,22 +41,42 @@ func (s *NodeScaleRKE1DownAndUp) SetupSuite() {
 	s.client = client
 }
 
-func (s *NodeScaleRKE1DownAndUp) TestEtcdScaleDownAndUp() {
-	s.Run("rke1-etcd-node-scale-down-and-up", func() {
-		ReplaceRKE1Nodes(s.T(), s.client, s.client.RancherConfig.ClusterName, true, false, false)
-	})
-}
-func (s *NodeScaleRKE1DownAndUp) TestWorkerScaleDownAndUp() {
-	s.Run("rke1-worker-node-scale-down-and-up", func() {
-		ReplaceRKE1Nodes(s.T(), s.client, s.client.RancherConfig.ClusterName, false, false, true)
-	})
-}
-func (s *NodeScaleRKE1DownAndUp) TestControlPlaneScaleDownAndUp() {
-	s.Run("rke1-controlplane-node-scale-down-and-up", func() {
-		ReplaceRKE1Nodes(s.T(), s.client, s.client.RancherConfig.ClusterName, false, true, false)
-	})
+func (s *RKE1NodeReplacingTestSuite) TestReplacingRKE1Nodes() {
+	nodeRolesEtcd := nodepools.NodeRoles{
+		Etcd:         true,
+		ControlPlane: false,
+		Worker:       false,
+	}
+
+	nodeRolesControlPlane := nodepools.NodeRoles{
+		Etcd:         false,
+		ControlPlane: true,
+		Worker:       false,
+	}
+
+	nodeRolesWorker := nodepools.NodeRoles{
+		Etcd:         false,
+		ControlPlane: false,
+		Worker:       true,
+	}
+
+	tests := []struct {
+		name      string
+		nodeRoles nodepools.NodeRoles
+		client    *rancher.Client
+	}{
+		{"Replacing RKE1 control plane nodes", nodeRolesControlPlane, s.client},
```

Comment:
supernit; the cluster type (rke1) is in the suite name, so we don't need it in the test name too.

Reply:
True, I had it just there so it's the same across the board since it looked a bit cleaner to me. I can remove them though, not a big deal.